modules/apps/change-tracking/change-tracking-service/src/main/java/com/liferay/change/tracking/internal/conflict/CTConflictChecker.java

```diff
@@ -1033,10 +1033,8 @@ private void _updateModelMvccVersion(
 
 					long mvccVersion = resultSet.getLong(2);
 
-					ctEntry.setModifiedDate(ctEntry.getModifiedDate());
-					ctEntry.setModelMvccVersion(mvccVersion);
-
-					ctEntry = _ctEntryLocalService.updateCTEntry(ctEntry);
+					ctEntry = _ctEntryLocalService.updateModelMvccVersion(
+						ctEntry.getCtEntryId(), mvccVersion);
```

Comment:
@david-truong is this correct? I believe `setModifiedDate` is redundant so I left it out. [_ctEntryLocalService.updateCTEntry](https://github.com/liferay/liferay-portal/blob/master/modules/apps/change-tracking/change-tracking-service/src/main/java/com/liferay/change/tracking/service/impl/CTEntryLocalServiceImpl.java#L283-L284) is only allowed for ctCollections that are in draft or pending status, so I tried to avoid it here to make the check work for scheduled publications.

Reply:
yes that should be okay.